src/filesystem/FileSystem.js

```diff
@@ -261,7 +261,8 @@ define(function (require, exports, module) {
     FileSystem.prototype._watchOrUnwatchEntry = function (entry, watchedRoot, callback, shouldWatch) {
         var impl = this._impl,
             recursiveWatch = impl.recursiveWatch,
-            commandName = shouldWatch ? "watchPath" : "unwatchPath";
+            commandName = shouldWatch ? "watchPath" : "unwatchPath",
+            ignored = watchedRoot.globFilter;
```

Comment:
I think we should consisten we the name. I see ignored, globFilter, globFilter.
Something like `ignoredFilters` would retain a similar name of chokidar option but a bit more descriptive.